src/ConfigResource.php

```diff
@@ -349,4 +353,15 @@ protected function invalidateCache($filename)
 
         opcache_invalidate($filename, true);
     }
+
+    protected function sortKeysRecursively(array &$data)
```

Comment:
Please add a docblock for this new method.

Reply:
Done, I've also removed the return statement since it didn't make sense.